src/xdocs/config_imports.xml

```diff
@@ -739,6 +749,19 @@ import android.*;
         </source>
 
         <p>
+          To configure the check to only check the &quot;src/test&quot; directory
```

Comment:
Example should probably be `src/main` as in romani's case, he wants to control imports in production code, and allow full range in test.